galata/test/jupyterlab/zoom.test.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+
+const sidebarIds: galata.SidebarTabId[] = [
+  'filebrowser',
+  'jp-property-inspector',
+  'jp-running-sessions',
+  'table-of-contents',
+  'extensionmanager.main-view'
+];
+
+test.use({ viewport: { width: 320, height: 256 } });
+
+test.describe('Low Vision / Zoom Support', () => {
+  sidebarIds.forEach(sidebarId => {
+    test(`Open Sidebar tab ${sidebarId}`, async ({ page }) => {
+      await page.sidebar.openTab(sidebarId);
+      expect(await page.sidebar.isTabOpen(sidebarId)).toEqual(true);
+    });
+  });
```

Comment:
This does not do belong here I think, the sidebars are tests elsewhere and checking that they can open does not appear relevant to low vision support

```suggestion
```